### PR TITLE
:ambulance: :books: Correcting raiseIntent examples

### PR DESCRIPTION
**Pull request by kriswest**
Addresses issue #210

### CODE REVIEW

File: src/api/DesktopAgent.ts [hunk 1]
```diff
@@ -102,10 +102,10 @@ interface DesktopAgent {
...
-     * const intentR = await agent.findIntents("StartChat", context);
-     * //use the IntentResolution object to target the same chat app with a new context
-     * agent.raiseIntent("StartChat", newContext, intentR.source);
+     * //Find apps to resolve an intent to start a chat with a given contact
+     * const appIntent = await fdc3.findIntent("StartChat", context);
+     * //use the returned AppIntent object to target one of the returned chat apps with the context
+     * await fdc3.raiseIntent("StartChat", context, appIntent.apps[0].name);
      * ```
      */
     raiseIntent(intent: string, context: Context, target?: string): Promise<IntentResolution>;
```
Comment (rikoe) [inline src/api/DesktopAgent.ts:111]: @kriswest Thanks for this! I think `Promise<IntentResolution>` should actually change too right?

Comment (kriswest) [reply src/api/DesktopAgent.ts:111]: @rikoe yes you are correct, will update PR

Comment (kriswest) [reply src/api/DesktopAgent.ts:111]: @rikoe I thought so at first, but no I think thats correct - rather the example was written as if `fdc3.findIntents()` would return an IntentResolution, when `fdc3.findIntent()` should return AppIntent.

Comment (rikoe) [reply src/api/DesktopAgent.ts:111]: Oh yes, I was getting confused with `findIntent` and `findIntentsByContext`, but this is `raiseIntent`...

**Pull request merged**